src/wasm-features.h

```diff
@@ -44,11 +44,13 @@ struct FeatureSet {
     ExtendedConst = 1 << 13,
     Strings = 1 << 14,
     MultiMemory = 1 << 15,
+    // FIXME(frank-emrich) Model dependencies on ref types and exceptions?
+    TypedContinuations = 1 << 16,
```

Comment:
I wonder whether we should change name to `wasmfx`... it is easier to type and catchier to pronounce (and more memorable!).